Makefile

```
@@ -0,0 +1,54 @@
+#### Variables ####
+
+export ROOT_DIR ?= $(PWD)
+export SRS_ROOT_DIR ?= $(ROOT_DIR)
+
+export ANSIBLE_NAME ?= ansible-srs
+export HOSTS_INI_FILE ?= hosts.ini
+
+export EXTRA_VARS ?= ""
+
+#### Start Ansible docker ####
+
+srs-ansible:
```

Same as previous comment. This comment is applicable across the repo
```suggestion
srsran-ansible:
```